third-culture-software/bhima
# Fix missing rates

If a new currency is added, urges the user to set it's exchange rate before doing anything.  It does this by popping up a modal urging them to add an exchange rate for the new currency/currencies.  It provides a like to the exchange page and adds a notice to the exchange setting modal about which currency needs to be added.

Closes https://github.com/IMA-WorldHealth/bhima/issues/5581

*TESTING*
- Use bhima_test
- Edit test/data.sql and comment out/remove the line in "INSERT INTO `exchange_rate` VALUES" for Euros.  In other words, delete this line:   (3, 1, @EUR, 0.8400, NOW())
- Rebuild the database:   yarn build:db
- Run Bhima
- Go to the top/main page and see the new modal
- Follow the link to the Exchange page and then try setting exchange rates
- Note the hint about which one needs setting
- Set the rate
- Verify things work normally now.
